### ISSUE TITLE
Generate the nix declarations by querying packages.edgedb.com

### ISSUE THREAD

**Issue by aljazerzen** [geldata/packages-nix#1]
Currently there is only a few edgedb-server versions declared in the flake.nix.

Ideally, we'd be able to use the source-of-truth for edgedb releases, packages.edgedb.com, to generate all available version derivations.

I'm not experienced enough with nix to do this in pure nix, so I'll probably fallback to using Python to do this. Does anyone have more experience?

**Comment by aljazerzen** [comment]
This is now mostly solved by 8f79363dca2bf29907d9e3439d700e2ae008d9d8

What remains is to make it truly automatic, so one can run a single command and have it be done without copy-pasting.

**Comment by darinkishore** [comment]
Could you see if the PR above [PR](https://github.com/edgedb/packages-nix/pull/2) does what you'd like it to? @aljazerzen

**Comment by gnull** [comment]
It seems that a CI action to periodically fetch updates and make a commit with new hashes would be most appropriate and idiomatic for this.

**Comment by aljazerzen** [comment]
Yeah, probably. There is `lookup_artifacts.py` that already does the fetch
